app/scripts/models/auth_brokers/fx-sync.js

```diff
@@ -12,14 +12,13 @@ define(function (require, exports, module) {
   'use strict';
 
   const BaseAuthenticationBroker = require('models/auth_brokers/base');
-  const { FX_SYNC_CONTEXT } = require('lib/constants');
 
   const proto = BaseAuthenticationBroker.prototype;
 
   module.exports = BaseAuthenticationBroker.extend({
     defaultBehaviors: proto.defaultBehaviors,
 
-    type: FX_SYNC_CONTEXT
+    type: 'fx-sync'
```

Comment:
Part of me wonders whether we should have some kind of client-side unit test that iterates over all the auth brokers and makes sure their `type` matches the validation regex. What do you think, worth it or a pointless test?